diff --git a/node_modules/@digitalbazaar/http-client/dist/cjs/index.cjs b/node_modules/@digitalbazaar/http-client/dist/cjs/index.cjs
index c81a996..97954cf 100644
--- a/node_modules/@digitalbazaar/http-client/dist/cjs/index.cjs
+++ b/node_modules/@digitalbazaar/http-client/dist/cjs/index.cjs
@@ -78,7 +78,7 @@ function deferred(f) {
  * Copyright (c) 2020-2022 Digital Bazaar, Inc. All rights reserved.
  */
 
-const kyOriginalPromise = deferred(() => import('ky-universal')
+const kyOriginalPromise = deferred(() => import('ky')
   .then(({default: ky}) => ky));
 
 const DEFAULT_HEADERS = {
