diff --git a/node_modules/ipfs-utils/src/http/fetch.js b/node_modules/ipfs-utils/src/http/fetch.js
index 1a27129..d1308b6 100644
--- a/node_modules/ipfs-utils/src/http/fetch.js
+++ b/node_modules/ipfs-utils/src/http/fetch.js
@@ -8,15 +8,15 @@
  * @property {globalThis.Headers} fetchImpl.Headers
  */
 
-let implName = './fetch.node'
-
-if (typeof XMLHttpRequest === 'function') {
-  // Electron has `XMLHttpRequest` and should get the browser implementation
-  // instead of node.
-  implName = './fetch.browser'
-}
-
-/** @type {fetchImpl} */
-const fetch = require(implName)
+// let implName = './fetch.node'
+//
+// if (typeof XMLHttpRequest === 'function') {
+//   // Electron has `XMLHttpRequest` and should get the browser implementation
+//   // instead of node.
+//   implName = './fetch.browser'
+// }
+//
+// /** @type {fetchImpl} */
+// const fetch = require(implName)
 
 module.exports = fetch
